src/data/blob.ts

```diff
@@ -34,7 +36,9 @@ export const parseFileContents = async (blob: Blob | File) => {
     }
     if (blob.type === "image/svg+xml") {
       try {
-        return await (await import("./image")).decodeSvgMetadata({
+        return await (
+          await import(/* webpackChunkName: "image-svg" */ "./image")
```

Comment:
```suggestion
          await import(/* webpackChunkName: "image" */ "./image")
```

this is the same chunk as for png so we shouldn't call it differently